notebooks/Liquidators_analysis.ipynb

```diff
@@ -0,0 +1,1351 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Group by 'User' and 'Protocol' to get the number of liquidators per protocol\n",
```

Comment:
Hint: Check the values of the `key_name` column in the data. You'll find some liquidation events there.